cellfinder/core/classify/classify.py

```diff
@@ -103,7 +103,7 @@ def main(
         points_list.append(cell)
 
     time_elapsed = datetime.now() - start_time
-    print(
+    logger.warning(
```

Comment:
Should this maybe be `logger.debug` or even `logger.info`? What made you choose `logger.warning` for this case?